### DISCUSSION TITLE
Cannot use a Github Maven package inside Gradle Kotlin

### DISCUSSION THREAD

**Discussion by anjoismysign** [community/community#46675] [Packages]
### Select Topic Area

Issue

### Body

So I am trying to use a package I published as a dependency in Gradle.
The issue is that whenever reloading Gradle, I receive the following message:

> Could not find me.anjoismysign.anjo:Anjo:.
     Required by:
         project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
 

 This is my current build.gradle.kts : 
plugins {
    id("java")
    id("org.jetbrains.intellij") version "1.12.0"
}

group = "us.mytheria"
version = "1.0.0"

repositories {
    mavenCentral()
    maven {
        url = uri("https://maven.pkg.github.com/anjoismysign/Anjo")
        credentials {
            username = project.findProperty("gpr.user") as String? ?: System.getenv("anjoismysign")
            password = project.findProperty("gpr.key") as String?
                    ?: System.getenv("ghp_SECRET!!!!!!!!!!!!")
        }
    }
}

dependencies {
    implementation("me.anjoismysign.anjo:anjo")
}

// Configure Gradle IntelliJ Plugin
// Read more: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html
intellij {
    version.set("2021.1.3")
    type.set("IC") // Target IDE Platform

    plugins.set(listOf(/* Plugin Dependencies */))
}

tasks {
    // Set the JVM compatibility versions
    withType<JavaCompile> {
        sourceCompatibility = "11"
        targetCompatibility = "11"
    }

    patchPluginXml {
        sinceBuild.set("221")
        untilBuild.set("231.*")
    }

    signPlugin {
        certificateChain.set(System.getenv("CERTIFICATE_CHAIN"))
        privateKey.set(System.getenv("PRIVATE_KEY"))
        password.set(System.getenv("PRIVATE_KEY_PASSWORD"))
    }

    publishPlugin {
        token.set(System.getenv("PUBLISH_TOKEN"))
    }
}

**Comment by wiefunkdai** [comment]
Yes.. Im problem same too, I fixed it by changing the entire java library structure to android library. It seems that the Android library package in Androidstudio is specifically only for the Android IDE, and because the Java Library can be accessed multiple times, this package is not available.

**Comment by Sneizzle** [comment]
Hey, I had a similar issue, and I fixed it by turning my Java library into an Android library. Android libraries work better with Android projects. If you want to use a Java library in an Android project, you can do that too. Just add it as a dependency in your Android project's settings, and it should work fine. Hope that helps!😊👍
